### DISCUSSION TITLE
Sort personal stats.

### DISCUSSION THREAD

**Discussion by BuddyBob** [monkeytypegame/monkeytype#1591] [Ideas] [answered]
When you click your account and scroll all the way down it shows you all of your entries. 
<img width="1140" alt="Screen Shot 2021-07-03 at 9 08 11 AM" src="https://user-images.githubusercontent.com/66140321/124360465-d7856b80-dbde-11eb-8e14-6affcf076b88.png">
**| wpm | raw | accuracy | etc.**
I suggest a couple of buttons to sort this data. 

<li> by time
<li> by accuracy
<li> by wpm
<li> by consistency

**Comment by Miodec** [comment, marked as answer]
Added issue #1602
